demos/Clamp/main.vala

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1
+
+public void main () {
+    var button_increase = (Gtk.Button) workbench.builder.get_object ("button_increase");
```

Comment:
Please make the buttons and clamp global variables instead of passing them as parameters, just as in the JS demo.